Source/EPUBCore/FRMediaType.swift

```diff
@@ -9,21 +9,21 @@
 import UIKit
 
 /**
-MediaType is used to tell the type of content a resource is.
+ MediaType is used to tell the type of content a resource is.
 
-Examples of mediatypes are image/gif, text/css and application/xhtml+xml
-*/
+ Examples of mediatypes are image/gif, text/css and application/xhtml+xml
+ */
 struct MediaType {
-    var name: String
-    var defaultExtension: String!
-    var extensions: [String]!
-    
+    var name                : String
+    var defaultExtension    : String!
+    var extensions          : [String]!
```

Comment:
FYI: In most cases you can see those attribute declarations in the diff due to a re-indentation or adding white spaces (which will be removed as requested).